### ISSUE TITLE
Auto view Reset on clicking create seeds

### ISSUE THREAD

**Issue by ashishsingh18** [dboun/MultiPatientImageProcessing#47]
Automatic View reset happens on clicking 'create seeds'. 
To reproduce, Zoom in Axial, Coronal and Sagittal views and then click on 'Create seeds'.

**Comment by dboun** [comment]
Yep, that definitely happens. It was happening on many other instances too, but I never managed to fix that one. It might be a limitation of MITK 2018.04. It also happens on workbench on similar scenarios
